canonical-web-and-design/maas-ui
# Only show release notifications on settings and machine list

## Done

- Limit the new release notifications to only show on the settings and machine list.
- Filter out release notifications from all legacy pages.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Create a new release notification (instructions are here: https://github.com/canonical-web-and-design/MAAS-squad/issues/2065#issue-669660220).
- View settings and the machine list, the release notification should be visible.
- View /kvm, the notification should be hidden.
- Visit any legacy page, the notification should be hidden.

## Fixes

Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/1506.
